src/main/java/org/nypl/findawayclientlibrary/DownloadService.java

```diff
@@ -33,11 +37,16 @@
   // so can do a search in log msgs for just this class's output
   private static String TAG;
 
+  public static final Integer CHAPTER_PART_DEFAULT = new Integer(0);
```

Comment:
Is it possible to use an enum for these constants instead?

Reply:
Sure, I'll think about it.  I don't love the way java does enums, and tend not to use them much in that language.